CONTRIBUTING.md

```diff
@@ -71,6 +71,20 @@
 - Thanks in advance for your patience as we review your contributions;
   we do appreciate them!
 
+# Coding Convention
+
+Use clang-format to format all source files (\*.h, \*.cc, \*.proto) to
+a consistent format. You should run clang-format on all source files
+before submitting a pull request::
+
+  $ apt-get install clang-format clang-format-6.0
+
+For convenience there is a format.py script in tools/ that can be used
+to clang-format all files within the repo::
+
+  $ cd .../triton-inference-server    # top-level of repo
+  $ python format.py *
```

Comment:
`python3` instead of `python`. It seems like `python` formats things differently.